keps/sig-node/4381-dra-semantic-parameters/README.md

```diff
@@ -259,23 +255,23 @@ demonstrate the interest in a KEP within the wider Kubernetes community.
 
 ### Publishing node resources
 
-The kubelet publishes NodeResourceSlices to the API server with content
-provided by DRA drivers running on its own node. It's also responsible for
-deleting or updating stale objects. The `nodeName` field determines which
-objects each kubelet instance is responsible for. Access control through the node
-authorizer ensures that the kubelet running on one node is not allowed to
-create or modify NodeResourceSlices belonging to other nodes.
+Each kubelet publishes a set of `NodeResourceSlice` objects to the API server
```

Comment:
Taking into consideration this [discussion](https://github.com/kubernetes/enhancements/pull/4384/files#r1481967588), I think we should change this as follows:

```
The resources available on a node need to be published to the API server. In the typical case, this is expected to be published by the on-node driver via the kubelet, as described below. However, the source of this data may vary; for example, a cloud provider controller could populate this based upon information from the cloud provider API.

In the kubelet case, each kubelet publishes...
```

Reply:
updated